src/config/logger.ts

```diff
@@ -67,7 +84,8 @@ const logger = wrapLogger(createLogger(
 		serializers: {
 			err: errorSerializer,
 			res: responseSerializer,
-			req: requestSerializer
+			req: requestSerializer,
+			...unsafeDataSerilaizers
```

Comment:
Hi @joshkay10 , it seems in the code we use a lot of `context.log` and `req.log`. Can you plz show me how are they configured to use this instance of `const logger = wrapLogger(...)`?